Translators/WeatherStations/ws_tim_translator.py

```diff
@@ -0,0 +1,84 @@
+import logging
+from Translators.Shared.active_tim import active_tim
+from Translators.WeatherStations.ws_tim_generator import get_geometry, get_itis_codes
+
+class WeatherStationFeature:
+    def __init__(self, properties, geometry):
+        self.id = properties["id"].replace("_", "-")
+        self.route = properties["routeName"].replace("_", "-")
+        self.geometry = geometry
+        self.direction = properties["direction"]
+        self.surface_status_sensor = [sensor for sensor in properties["sensors"] \
+                                        if sensor["type"] is not None and sensor["type"].lower() == "road surface status"]
+        self.wind_gust_sensor = [sensor for sensor in properties["sensors"] \
+                                        if sensor["type"] is not None and sensor["type"] == "gust wind speed"]
+        self.avg_wind_speed_sensor = [sensor for sensor in properties["sensors"] \
+                                        if sensor["type"] is not None and sensor["type"] == "average wind speed"]
+
+    def get_id(self):
+        return self.id
+    
+    def get_client_id(self):
+        return self.id.replace("/", "-")
```

Comment:
question: is this client id guaranteed to be the same for the same condition? IE if we get wind gust speed over our 20, will that be the same id the next time we see it here?

Reply:
Yes, the client ID will be the same for all future conditions

Reply:
Perfect, thanks!